Makefile

```
@@ -23,6 +23,14 @@ gold:
 fmt:
 	dune build @fmt --auto-promote
 
+# Update src/ParserErrors.messages
+update-messages:
+	menhir --list-errors src/Parser.mly >src/NewParserErrors.messages
+	menhir --merge-errors src/ParserErrors.messages \
+		   --merge-errors src/NewParserErrors.messages \
+		   src/Parser.mly  >src/NewParserErrors.messages
+	mv src/NewParserErrors.messages src/ParserErrors.messages
```

Menhir does no cope well with reading and writing to the same file, here is a suggested change:
```suggestion
	menhir --list-errors src/Parser.mly >src/NewParserErrorsStubs.messages
	menhir --merge-errors src/ParserErrors.messages \
		   --merge-errors src/NewParserErrorsStubs.messages \
		   src/Parser.mly  >src/NewParserErrors.messages
	mv src/NewParserErrors.messages src/ParserErrors.messages
	rm src/NewParserErrorsStubs.messages
```